environment.yml

```
@@ -0,0 +1,6 @@
+name: django-asv
+dependencies:
+  - pip
+  - wheel
+  - conda-build
+  - libmambapy
```

```suggestion
  - conda-build
  - libmambapy
  - pip
  - wheel
```
I only sorted dependencies, it can help in managing them.